files/rsyslog.conf

```diff
@@ -0,0 +1,4 @@
+$FileCreateMode 0644
+
+:syslogtag, startswith, "pollen" /var/log/pollen/pollen.log
+& ~
```

Comment:
Missing blank line at EOF

Reply:
Still seems to be there?

Reply:
As with the other comments, I forgot to upload the changes after resolving the comments. Now you can take a look at the PR with the original comments addressed.